docs/Development-Guides/Rate-Limits.md

```diff
@@ -2,7 +2,10 @@
 
 Finch will return a rate limit error with the HTTP status code 429 when the request rate limit for an application, access token, or an individual IP address has been exceeded.
 
-Finch's rate limits work on a per-product basis for both applications and access tokens. Rate limits are summed on a rolling 60-second basis for each unique `product`. This is commonly referred to as a Sliding or Rolling Window rate limit.
+Finch's rate limits work on a per-product basis for applications. Rate limits are summed on a rolling 60-second basis for each unique `product`. This is commonly referred to as a Sliding or Rolling Window rate limit.
```

Comment:
I'm not sure of the desired nomenclature but "per-product" might be contextually easier to understand if we just said "per-endpoint" or "per-path".

Reply:
Good catch, this was confusing and I've updated it.